client/src/includes/initIconSprite.ts

```diff
@@ -0,0 +1,51 @@
+export const initIconSprite = (
+  spriteContainer: HTMLElement,
+  spriteURL: string,
+  // eslint-disable-next-line consistent-return
+) => {
+  const revisionKey = 'wagtail:spriteRevision';
+  const dataKey = 'wagtail:spriteData';
+  const isLocalStorage: boolean =
+    'localStorage' in window && typeof window.localStorage !== 'undefined';
+  const insertIt = (data: string) => {
+    if (spriteContainer) {
+      // eslint-disable-next-line no-param-reassign
+      spriteContainer.innerHTML = data;
+    }
+  };
+
+  const insert = (data: string) => {
+    if (document.body) {
+      insertIt(data);
+    } else {
+      document.addEventListener('DOMContentLoaded', () => insertIt(data));
+    }
+  };
+
+  if (isLocalStorage && localStorage.getItem(revisionKey) === spriteURL) {
+    const data = localStorage.getItem(dataKey);
+    if (data) {
+      insert(data);
+      return true;
```

Comment:
```suggestion
      return;
```

This function can probably just be a void function - the return value is not used. This way you can remove the Eslint ignoring above also.